### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by cocainomano** [Yooooomi/your_spotify#418] [Q&A]
Hello there, 

Im using Yooomi with docker. I'm not an expert but an enthusiastic. When I installed Yooooomi, it took me quite a while because I had never used Docker before. It was a pleasant experience for me, and I learned a lot, but I don't think I learned enough to update the containers and the database without messing everything up. Is there any guide or advice on how to update the version? Thanks, everyone.

**Comment by zabit-z** [comment]
use `docker pull` for update
